README.md

```diff
@@ -60,6 +60,7 @@ NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 foo-pravega-operator     1         1         1            1           17s
 ```
 
+> Note:  We can run operator in testmode by passing `testmode: true` in `values.yaml` file. This will be useful if we want to run pravega on minikube or cluster with limited resources. In test mode, operator will allow us to run pravega with single bookkeeper instance.
```

Comment:
It would be good to start like : "The Operator can be run in a "test" mode if we want to create pravega on minikube or on a cluster with very limited resources." Also clearly state what the **test** mode does
"
Operator running in test mode skips minimum replica requirement checks on Pravega components.
" 
It would be good to add a warning that this is a bare minimum setup and not **recommended** to be used in production environments.

Reply:
Fixed.